.azure-pipelines/recover_testbed/common.py

```diff
@@ -71,6 +71,14 @@ def posix_shell_onie(dut_console, mgmt_ip, image_url, is_nexus=False):
                         dut_console.remote_conn.send('reboot\n')
                         continue
 
+                    if is_nokia and enter_onie_flag is True:
```

Comment:
Have you figured out whether the sequence and priority affect boolean usage?

Reply:
This sequence will work as expected